### DISCUSSION TITLE
Chia has a good idea

### DISCUSSION THREAD

**Discussion by slebedev1976** [Chia-Network/chia-blockchain#3393] [General]
With the Chia cryptocurrency, SSD manufacturers will be able to make a significant leap forward in data storage. I hope that our joint investment in this cryptocurrency will allow many of them to do this. And for us, ordinary people, our investments will pay off like Bitcoin.

**Comment by pluto2021** [comment]
Nothing goes only up. Crypto is violently volatile. So get ready for something unexpected and do not panic!
